aries-site/src/data/structures/components.js

```diff
@@ -944,6 +944,7 @@ export const components = [
   },
   {
     name: 'Grid',
+    accessibility: 'Passed WCAG 2.2 AA',
```

Comment:
Looks like we are meeting AAA
```suggestion
    accessibility: 'Passed WCAG 2.2 AAA',
```